guides/common/modules/proc_configuring-the-active-directory-authentication-source-on-projectserver.adoc

```diff
@@ -0,0 +1,58 @@
+[id="configuring-the-active-directory-authentication-source-on-projectserver_{context}"]
+= Configuring the Active Directory authentication source on {ProjectServer}
+
+Enable Active Directory (AD) users to access {Project} by configuring the corresponding authentication provider on your {ProjectServer}.
+
+.Prerequisites
+* The base system of your {ProjectServer} must be joined to an Active Directory (AD) domain.
+For more information, see link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/8/html-single/integrating_rhel_systems_directly_with_windows_active_directory/index#connecting-rhel-systems-directly-to-ad-using-sssd_integrating-rhel-systems-directly-with-active-directory[Connecting RHEL systems directly to AD using SSSD].
+
+.Procedure
+. Configure the System Security Services Daemon (SSSD) to use the AD access control provider to evaluate and enforce Group Policy Object (GPO) access control rules for the `foreman` PAM service:
+.. In your `/etc/sssd/sssd.conf` file, add the following lines to the `domain` section for the AD domain:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+[domain/_ad.example.com_]
+ad_gpo_access_control = enforcing
+ad_gpo_map_service = +foreman
+----
+ifndef::orcharhino[]
++
+For more information on GPOs, see link:{RHELDocsBaseURL}9/html/integrating_rhel_systems_directly_with_windows_active_directory/managing-direct-connections-to-ad_integrating-rhel-systems-directly-with-active-directory#how-sssd-interprets-gpo-access-control-rules_applying-group-policy-object-access-control-in-rhel[How SSSD interprets GPO access control rules] in _{RHEL}{nbsp}9 Integrating RHEL systems directly with Windows Active Directory_ or link:{RHELDocsBaseURL}8/html/integrating_rhel_systems_directly_with_windows_active_directory/managing-direct-connections-to-ad_integrating-rhel-systems-directly-with-active-directory#applying-group-policy-object-access-control-in-rhel_managing-direct-connections-to-ad[How SSSD interprets GPO access control rules] in _{RHEL}{nbsp}8 Integrating RHEL systems directly with Windows Active Directory_.
+endif::[]
+.. Restart SSSD:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# systemctl restart sssd
+----
+. Configure AD as the authentication source for your {ProjectServer}:
+.. Create an Apache keytab file if it does not already exist on your system:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# touch /etc/httpd/conf/http.keytab
+----
++
+You can leave the file empty.
+The `{foreman-installer}` command fails to configure the authentication source if the file is missing.
+.. Enable IPA authentication:
```

Comment:
It's the first time that I read "IPA" in this section.

* Should it be expanded?
* Section 5.4 mentions "Identity Management" as an alternative to direct AD integration. Why now IPA? If it doesn't matter in the background if IPA or AD, can you rephrase the text?

Reply:
This is tricky. I kept "IPA" literal and un-expanded on purpose. "Identity Management" is not part of this scenario so I can't use that. The `--foreman-ipa-authentication` option is used when connecting to an IdM identity store (indirect integration, as described in section 5.4), but also for the SSSD/AD scenarios (direct integration, as described in this PR). I assume that's because in both indirect and direct scenarios, `--foreman-ipa-authentication` actually configures an SSSD client. So in a sense, when I decided to go with "enable IPA authentication", I just translated the `--foreman-ipa-authentication` option's name.

I think I can go with something generic like "Enable the authentication source". You are right that talking about "IPA" would raise too many questions.